### ISSUE TITLE
MailChimp subscribe not working

### ISSUE THREAD

**Issue by jooncheez** [kelyvin/caffeine-theme#28]
I try to add MailChimp subscribe to my website. 

This is what I add, exactly the same format as the example:

```
<script>
var mailchimpOptions = {
    url: "//janvdkolk.us14.list-manage1.com/subscribe?u=18abd455664ef9cbac66fd316&id=ac22da35f1"
};
</script>
```

The subscribe button appears, but after I fill in name and email and click send, nothing happens. The link works, you can check it.

When I try the code from the example it does work and I receive a confirmation email.

**Comment by kelyvin** [comment]
Hi @jooncheez, 

You are just missing one part of the url. The subscribe feature makes an HTTP `POST` request to the mailchimp API to send the subscription request. 

So modify your mailchimp url so that it looks like the following:
`//janvdkolk.us14.list-manage1.com/subscribe/post?u=18abd455664ef9cbac66fd316&id=ac22da35f1`

The only change I made was that `/post` was added right after the `/subscribe`. Give that a try and let me know if that works for you.
